arches/app/datatypes/datatypes.py

```diff
@@ -2622,6 +2622,16 @@ def clean(self, tile, nodeid):
     def transform_export_values(self, value, *args, **kwargs):
         new_values = value
         return ",".join(new_values)
+    
+    def get_display_value(self, tile, node, **kwargs):
+        labels = []
+        requested_language = kwargs.pop("language", None)
+        current_language = requested_language or get_language()
+        for item in self.get_tile_data(tile)[str(node.nodeid)]:
+            for label in item["labels"]:
+                if label["language"] == current_language and label["valuetype"] == "prefLabel":
```

Comment:
This is okay for now, but in the next iteration we could probably use `rank_label()` so that we have good fallbacks. Right now it will fallback to nothing.